charts/argo-cd/templates/argocd-notifications/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+{{- $config := .Values.notifications.clusterAdminAccess | default dict -}}
+{{- if hasKey $config "enabled" | ternary $config.enabled .Values.createClusterRoles }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "argo-cd.notifications.fullname" . }}
+  labels:
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.notifications.name "name" .Values.notifications.name) | nindent 4 }}
+rules:
+  {{- if .Values.notifications.clusterRoleRules.enabled }}
+    {{- toYaml .Values.notifications.clusterRoleRules.rules | nindent 2 }}
+  {{- else }}
```

Comment:
maybe we don't need `enabled` flag ? 🙋 

```yaml
{{- with.Values.notifications.clusterRoleRules.rules }}
  {{- toYaml . | nindent 2 }}
{{- end }}
```

Reply:
Thats fair, was copy pasted

Reply:
Fixed, plus removed deprecated syntax/vars